Fix Display Owner Error

Caused by: org.attoparser.ParseException: Exception evaluating SpringEL expression: "telephone" (template: "owner/ownerDetails" - line 27, col 13)
	at org.attoparser.MarkupParser.parseDocument(MarkupParser.java:393)
	at org.attoparser.MarkupParser.parse(MarkupParser.java:257)
	at org.thymeleaf.templateparser.markup.AbstractMarkupTemplateParser.parse(AbstractMarkupTemplateParser.java:230)
	... 55 more

--- a/pet-clinic-data/src/main/java/com/springboot/petclinic/entity/PetType.java
+++ b/pet-clinic-data/src/main/java/com/springboot/petclinic/entity/PetType.java
@@ -18,4 +18,8 @@ public class PetType extends BaseEntity{
     @Column(name = "name")
     private String name;
 
+    @Override
+    public String toString() {
+        return name;
+    }
 }

--- a/pet-clinic-web/src/main/resources/templates/owner/ownerDetails.html
+++ b/pet-clinic-web/src/main/resources/templates/owner/ownerDetails.html
@@ -24,7 +24,7 @@
       </tr>
       <tr>
         <th>Telephone</th>
-        <td th:text="*{telephone}"></td>
+        <td th:text="*{phoneNumber}"></td>
       </tr>
     </table>
   
@@ -49,7 +49,7 @@
             <dd
               th:text="${#temporals.format(pet.birthDate, 'yyyy-MM-dd')}"></dd>
             <dt>Type</dt>
-            <dd th:text="${pet.type}"></dd>
+            <dd th:text="${pet.petType}"></dd>
           </dl>
         </td>
         <td valign="top">

